libs/coin-modules/coin-bitcoin/src/signOperation.ts

```diff
@@ -81,56 +82,100 @@ export const buildSignOperation =
 
         const hasExtraData = perCoin?.hasExtraData || false;
 
-        const signature: string = await signerContext(deviceId, currency, signer =>
-          wallet.signAccountTx({
-            btc: signer,
-            fromAccount: walletAccount,
-            txInfo,
-            lockTime,
-            sigHashType,
-            segwit,
-            additionals,
-            expiryHeight,
-            hasExtraData,
-            onDeviceSignatureGranted: () =>
-              o.next({
-                type: "device-signature-granted",
-              }),
-            onDeviceSignatureRequested: () =>
-              o.next({
-                type: "device-signature-requested",
-              }),
-            onDeviceStreaming: ({ progress, index, total }) =>
-              o.next({
-                type: "device-streaming",
-                progress,
-                index,
-                total,
-              }),
-          }),
-        );
-        // Build the optimistic operation
-        const operation: Operation = {
-          id: encodeOperationId(account.id, "", "OUT"),
-          hash: "", // Will be resolved in broadcast()
-          type: "OUT",
-          value: new BigNumber(transaction.amount).plus(fee),
-          fee,
-          blockHash: null,
-          blockHeight: null,
-          senders: Array.from(senders),
-          recipients,
-          accountId: account.id,
-          date: new Date(),
-          extra: {},
-        };
-        o.next({
-          type: "signed",
-          signedOperation: {
-            operation,
-            signature,
-          },
-        });
+        if (transaction.psbt) {
+          const psbtBuffer = Buffer.from(transaction.psbt, "base64");
+
+          const psbtResult = await signerContext(deviceId, currency, signer =>
+            signer.signPsbtV2Buffer
+              ? signer.signPsbtV2Buffer(psbtBuffer, {
+                  // @ts-expect-error: x
+                  finalizePsbt: transaction.finalizePsbt,
+                  accountPath: `${walletAccount.params.path}/${walletAccount.params.index}'`,
+                  addressFormat: getAddressFormatDerivationMode(account.derivationMode),
+                })
+              : Promise.reject(new Error("signPsbtV2Buffer not available")),
+          );
+
+          o.next({ type: "device-signature-granted" });
+
+          // Build the optimistic operation pour PSBT
+          const operation: Operation = {
+            id: encodeOperationId(account.id, "", "OUT"),
+            hash: "",
+            type: "OUT",
+            value: new BigNumber(transaction.amount).plus(fee),
+            fee,
```

Comment:
Calculate fees might not work as expected with a raw transaction as it uses buildTransaction to calculate the fees, we might want to look into it

Did you test the sync and make sure the history is synced correctly ?